src/components/ManageCookies.spec.tsx

```diff
@@ -20,18 +20,64 @@ describe('ManageCookies', () => {
     });
   });
 
-  describe('with osano', () => {
-    let showDrawerSpy: jest.SpyInstance;
+  describe('when CookieYes loads', () => {
+    const bannerLoadEvent = new CustomEvent('cookieyes_banner_load', {});
+    let component: renderer.ReactTestRenderer;
 
+    beforeEach(() => {
+      jest.clearAllMocks();
+    });
+
+    it('renders button', () => {
+      act(() => {
+        component = renderer.create(
+          <ManageCookiesLink />
+        );
+      });
+      act(() => { document.dispatchEvent(bannerLoadEvent); });
```

Comment:
I don't really understand why I had to do 2 separate `act`s here. I thought it should work with `renderer.create()` not in `act`. But unless I wrap `renderer.create()` in its own `act`, the component never renders.

Reply:
I think you need that `act` to make sure that hooks and state changes finish before the test runs. otherwise cookieyes never loads and the component renders null.